Tasks/UseDotNetV2/task.json

```diff
@@ -22,7 +22,7 @@
     "demands": [],
     "minimumAgentVersion": "2.144.0",
     "ecosystem": "dotnet",
-    "instanceNameFormat": "Use .Net Core $(packageType) $(version)",
+    "instanceNameFormat": "Use .NET Core $(packageType) $(version)",
```

Comment:
@NenoLoje please update the task patch version in task.json and task.loc.json

Reply:
Thanks @bishal-pdMSFT. Done!